Gordon360/Models/ViewModels/ApartmentApplicationViewModel.cs

```diff
@@ -9,12 +10,32 @@ public class ApartmentApplicationViewModel
         public int ApplicationID { get; set; }
         public DateTime? DateSubmitted { get; set; } // Nullable
         public DateTime DateModified { get; set; }
-        public string EditorUsername { get; set; }
-        public string EditorEmail { get; set; }
-        public string Gender { get; set; }
+        public PublicStudentProfileViewModel EditorProfile { get; set; }
+        private string _editorUsername;
+        public string EditorUsername
+        {
+            get => EditorProfile?.AD_Username ?? _editorUsername;
+            set => _editorUsername = value;
+        }
+        public string EditorEmail => EditorProfile?.Email;
+        public string Gender => EditorProfile?.Gender ?? Applicants?.First()?.Profile?.Gender;
```

Comment:
And it's guaranteed that the editor will be the first applicant in the list, right?

Reply:
Well all applicants are the same gender, by the rules of the application

Reply:
It is completely unnecessary, but I put it there just in case

It would be impossible to not have an editor, in theory

Reply:
Oh, whoops, I meant to comment on the line above this- it is a little more important that the email we show is definitely the editor's